os/arch/arm/src/armv8-m/mpu.h

```diff
@@ -286,6 +286,30 @@ static inline void mpu_showtype(void)
 #endif
 }
 
+/****************************************************************************
+ * Name: mpu_get_access_type
```

Comment:
Same as above. Please check the function name.

Reply:
ok.